paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -153,9 +154,11 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
 
     private fun createFragmentConfig(): FragmentConfig? {
         val stripeIntentValue = stripeIntent.value
-        val paymentMethodsValue = paymentMethods.value
         val isGooglePayReadyValue = isGooglePayReady.value
         val savedSelectionValue = savedSelection.value
+        // List of Payment Methods is not passed in the config but we still wait for it to be loaded
+        // before adding the Fragment.
+        val paymentMethodsValue = paymentMethods.value
```

Comment:
do we still need to check if nullability here? it seems when read this value we're using `orEmpty()` anyway

Reply:
I'm reading with `orEmpty()` to simplify the code, but the value is null until we've gone through `updatePaymentMethods()` and not null after that, indicating that they were already loaded (even if no payment method was loaded, we set the value to empty). Added a comment to `paymentMethods`.